components/DynamicForm/MyResult.tsx

```diff
@@ -56,9 +56,16 @@ function _updateTextInputs(
 ) {
   const formValues: FormValues = ctx.formValues
   const userInput: string = event.target.value
-  formValues.formAnswers[questionId].userAnswer = userInput
-  if (ctx.updateFormValues) {
-    ctx.updateFormValues(formValues)
+  if (ctx.setFormValues) {
```

Comment:
You can switch back to the 
`formValues.formAnswers[questionId].userAnswer = userInput`
and just call set.FormValues(formValues)

If there's some error that's happening because of mutation, tho then keep it